text/0000-api-version-compat.md

```diff
@@ -0,0 +1,119 @@
+# Meta
+[meta]: #meta
+- Name: Lifecycle API version support changes
+- Start Date: 2020-04-14
+- Author(s): [Emily Casey](https://github.com/ekcasey)
+- RFC Pull Request: (leave blank)
+- CNB Pull Request: (leave blank)
+- CNB Issue: (leave blank)
+
+# Summary
+[summary]: #summary
+
+In order enable smoother migrations and upgrades this RFC proposes the following changes to the API version compatibility rules:
+
+1. When the lifecycle increments an API versions in the `0.x` version line, this shall interpreted as non-breaking
+1. The spec may make certain types of "breaking" changes in a `0.x` API version, so long as those changes do not preclude simultaneous lifecycle support for the previous minor or require collaborating buildpacks to implement the same API version.
+1. When the lifecycle implments a new `0.x` API minor version, it will treat the usage of features removed between minor spec versions as a deprecated.
```

Comment:
```suggestion
1. When the lifecycle implments a new `0.x` API minor version, it will treat the usage of features removed between minor spec versions as deprecated.
```